homeassistant/components/google_assistant/trait.py

```diff
@@ -248,7 +249,10 @@ async def execute(self, command, data, params, challenge):
             self.state.entity_id, "hls"
         )
         self.stream_info = {
-            "cameraStreamAccessUrl": self.hass.config.api.base_url + url
+            "cameraStreamAccessUrl": async_get_url(
```

Comment:
I don't remember if `require_ssl=True` is necessary.

Reply:
Googled around, it seems it doesn't. The documentation doesn't mention anything about such a required and found quite a bit examples around the net with an `http://` link in an example. Nice!